data/lang/ui-core/en.json

```diff
@@ -1279,14 +1263,6 @@
     "description": "",
     "message": "Lobby"
   },
-  "LOCAL_ARM": {
-    "description": "Arm of the Milky Way galaxy",
-    "message": "Local arm"
-  },
-  "LOCATED_N_KM_FROM_THE_CENTRE_OF_NAME": {
```

Comment:
Keep this, and all strings mentioning ARM

Reply:
OK

Reply:
Fixed!